service/history/queues/priority_assigner.go

```diff
@@ -47,15 +47,23 @@ func NewPriorityAssigner() PriorityAssigner {
 }
 
 func (a *priorityAssignerImpl) Assign(executable Executable) tasks.Priority {
-	switch executable.GetType() {
+	taskType := executable.GetType()
+	switch taskType {
 	case enumsspb.TASK_TYPE_DELETE_HISTORY_EVENT,
 		enumsspb.TASK_TYPE_TRANSFER_DELETE_EXECUTION,
```

Comment:
This is generated from DeleteWorkflowExecution API. I think this should be higher priority.

Reply:
I take this back. Low priority for this should be fine.